app/src/main/res/values/strings.xml

```diff
@@ -428,7 +428,7 @@
   <string name="notifications">Notifications</string>
   <string name="read_notifications">Notifications (read)</string>
   <string name="display_nearby_notification">Display nearby notification</string>
-  <string name="display_nearby_notification_summary">Tap here to see the nearest place that needs pictures</string>
+  <string name="display_nearby_notification_summary">Be notified of nearest place that needs pictures</string>
```

Comment:
"Be notified" is slightly misleading as the user could easily expect to get a casual Android notification for this. While in reality, we only show in-app notifications. So, I think re-phrasing this as

> Show in-app notification for the nearest place that needs pictures

would be more precise.

Reply:
Sounds great. Go for it (make the edit, please.)
Also maybe "place" should be "places" unless really just one.

Reply:
>  Also maybe "place..needs" should be "places...need" unless really just one.

I've only ever seen 1 notification about a nearby place needing an image. So, I believe "placed ... need" is appropriate here.

> In the meantime, please make the changes for me. Thanks!

Sure. :)

Reply:
! Alas, GitHub doesn't have a friendly button to do that.

There's actually a way to suggest changes in GitHub in a way that should be easy to apply them. I just didn't think of using it when reviewing. That said, I've done the changes for this PR. For the other PRs, I'll suggest changes using the mentioned method so that you could easily apply them. 👍